### PR TITLE
Use proper pathname in checkPath

### PR DESCRIPTION
**Pull request by karelhala**
### CheckPath returning correct route

When calling `checkPath` it uses `location.pathname` which is incorrect. Let's use the `pathname` from react-router.

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -30,7 +32,7 @@ const App: React.FC = () => {
 
   return (
     <React.Fragment>
-      <NotificationsPortal />
+      <NotificationsPortal store={store} />
```
Comment (andrewballantyne) [inline frontend/src/App.tsx:35]: You up versioned our dependency and passed the store, help me understand what this does for us (or the chrome)?

**Review by andrewballantyne — COMMENTED** [review]
Looks great, I'm ready to approve, just one question...

File: frontend/src/App.tsx (hunk 1 above)
Comment (karelhala) [reply frontend/src/App.tsx:35]: There's a problem with shared react-redux in a couple of our components where they can't access directly the redux provider.

The version update was just so we are up to date with the config repo, there's been just a few adjustments to how files are split.

Comment (andrewballantyne) [reply frontend/src/App.tsx:35]: Interesting, we may need to look at how the SDK will interact with the Chrome in the long run... but this works for now. Thanks for the explanation! 👍

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**